tests/Profile/ServiceTest.php

```diff
@@ -133,6 +134,29 @@ public function testInvalidConnectToken()
         $profileService->getActivityDetails(TestData::INVALID_YOTI_CONNECT_TOKEN);
     }
 
+    /**
+     * Test invalid Token
+     *
+     * @covers ::getActivityDetails
+     * @covers ::decryptConnectToken
+     */
+    public function testWrongPemFile()
+    {
+        $this->expectException(\Yoti\Exception\ActivityDetailsException::class);
+        $this->expectExceptionMessage('Could not decrypt connect token');
```

Comment:
I know you're just testing for this exception message here, and you haven't added it, but "Connect" isn't something which is known outside of Yoti (I know we have `YOTI_CONNECT_TOKEN` and `INVALID_YOTI_CONNECT_TOKEN` too).
So I don't think this is a very helpful error message to anyone outside of Yoti. 

Could we instead refer to it as the one time use token, as we do in the README and documentation?

Reply:
Re-worded in e804d5f (I've left the test data as is)